package/docs/using-data/2. wizard.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Using the wizard helper
+description: The wizard helper makes it easy to build and iterate a ‘one thing per page’ user journey.
+tags: Getting started
+---
+
+## Importing the wizard helper
+
+```js
+import { wizard } from 'govuk-prototype-rig'
+```
+
+## Creating a list of paths
+
+```js
+const paths = [
+  '/examples/wizard',
+  '/examples/wizard/name',
+  '/examples/wizard/where-do-you-live',
+  '/examples/wizard/england',
+  '/examples/wizard/nationality',
+  '/examples/wizard/check-answers',
+  '/examples/wizard/confirm',
+  '/'
+]
+
+const nextAndBackPaths = wizard.nextAndBackPaths(paths, req)
+```
+
+## Creating a list of forks
+
+```js
+const forks = [{
+  currentPath: '/examples/wizard/where-do-you-live',
+  storedData: ['wizard', 'where-do-you-live'],
+  excludedValues: ['England'],
+  forkPath: '/examples/wizard/nationality'
+}]
+
+const nextForkPath = wizard.nextForkPath(forks, req)
+```
+
+Each fork accepts the following values:
+
+| Name | Type | Description |
+| :--- | :--- | :---------- |
+| **currentPath** | string | **Required.** Path to fork on. |
+| **forkPath** | string/Function | **Required.** Path to fork to based on answer. |
+| **storedData** | string/Array | **Required.** Data to check values against. |
+| **values** | string/Array | Values to include. |
+| **includedValues** | string/Array | Values to exclude. |
```

Comment:
```suggestion
| **excludedValues** | string/Array | Values to exclude. |
```

Reply:
Oooh, I think I changed this value on our prototype – will reinstate your original name.